k8s/aat/common/private-law/prl-dgs.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: prl-dgs
+  namespace: private-law
+spec:
+  releaseName: prl-dgs
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/prl-dgs
+  values:
+    java:
+      replicas: 2
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      # image: hmctspublic.azurecr.io/prl/dgs: #{"$imagepolicy": "flux-system:prl-dgs"}
```

Comment:
image tag is commented .

Reply:
There isn't yet a production image for dgs-api so I havent added it yet